### DISCUSSION TITLE
Creation of Student account keeps being rejected

### DISCUSSION THREAD

**Discussion by MargauxThirion** [community/community#52374] [GitHub Education]
Despite several tries, my account to GithHub education is systematically rejected by the portal. I inserted my university email address as well as a picture of my Student ID Card, valid for 2022/2023, but my applcation keeps being rejected.
Does the document has to be in english ?

**Comment by Ntshangase** [comment]
Hi

Translating the document does help the reviewers in verifying your application.

**Reply by MargauxThirion** [reply]
already tried but no more success... is there any solution to send them the official copies of the docs ?

**Reply by Ntshangase** [reply]
Is this your first time applying or you are renewing?

**Comment by samus-aran** [comment]
Hi @MargauxThirion 👋 

Here are a few tips on submitting evidence:

We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. Also consider that your evidence can be associated with the school you selected - does it have a readable logo, legible text, etc.

We also do accept multiple pieces of evidence in a single submission if you have this. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope this helps! Let us know how you get along 👍
